### ISSUE TITLE
Configuration.yaml contains !secret values

### ISSUE THREAD

**Issue by SteveDinn** [keesschollaart81/vscode-home-assistant#33]
I recently introduced a few !secret values into my configuration.yaml file, and I started getting this error from the extension:
![image](https://user-images.githubusercontent.com/3911507/57642420-a7595980-758d-11e9-9589-44ad1c5a406d.png)

I'm not sure if this is the same as another issue I saw about !includes, but my !includes seem to be fine. My configuration.yaml is pretty bare-bones, as most of my components are in packages:
```
homeassistant:
  # Name of the location where Home Assistant is running
  name: Home
  
  auth_providers:
    - type: homeassistant
    # uncomment this to enable backwards compatible API password support
    #- type: legacy_api_password

  # Location required to calculate the time the sun rises and sets
  latitude: !secret home_latitude
  longitude: !secret home_longitude
  
  # Impacts weather/sunrise data (altitude above sea level in meters)
  elevation: 0
  
  # metric for Metric, imperial for Imperial
  unit_system: metric
  
  # Pick yours from here: http://en.wikipedia.org/wiki/List_of_tz_database_time_zones
  time_zone: America/Halifax
  
  # Customization file
  customize: !include customize.yaml

  # Packages
  packages: !include_dir_named packages

http:
  # api_password: !secret http_password
  base_url: !secret base_url
  use_x_forwarded_for: true
  trusted_proxies:
    - 127.0.0.1

api:

person:
```

**Comment by keesschollaart81** [comment]
Hehe, I had a hard time figuring out why it failed because this is completely valid YAML of course :-). It is because it needs a whiteline under `person:`...

In the dev-branch I already have a big change prepared with a complete rewrite on the YAML parsing which fixes this. For now, add a whiteline and a fix will be rolling out later this week!

**Comment by DavidFW1960** [comment]
Similar errors here in mine https://github.com/DavidFW1960/home-assistant/blob/master/configuration.yaml

I get a validation error for elevation: 
```
  elevation: !secret my_elevation
```
And also my network auth:
```
# Auth & 2FA
  auth_mfa_modules:
    - type: notify
    - type: totp
  auth_providers:
    - type: trusted_networks
      trusted_networks:
        - 127.0.0.1
        - ::1
        - 10.90.11.0/24
        - fd00::/8
        - !secret zerotier_iphone
        - !secret zerotier_ipad
        - !secret my_ipv6_network
      trusted_users:
        127.0.0.1: !secret user_id
        "::1": !secret user_id
        10.90.11.0/24: !secret user_id
        "fd00::/8": !secret user_id
        !secret zerotier_iphone: !secret user_id
        !secret zerotier_ipad: !secret user_id
        !secret trusted_ipv6: !secret user_id
      allow_bypass_login: true
    - type: homeassistant
```
With the trusted user config

**Comment by DavidFW1960** [comment]
![image](https://user-images.githubusercontent.com/36390762/57667034-82c0b880-7645-11e9-8906-cc24f2ce6171.png)

**Comment by keesschollaart81** [comment]
Fix the `trusted_users` schema in the dev branch, will roll out this week!

**Comment by DavidFW1960** [comment]
That won't fix the elevation also being a secret value..

**Comment by keesschollaart81** [comment]
You're right but that was already fixed in 1f479cf03b59b6e1a6ece316dd992be4b20e523a. I'll push it out later this week!

**Comment by DavidFW1960** [comment]
sweet!
